src/types.ts

```diff
@@ -19,9 +19,8 @@ export interface Configuration<
   suspense: boolean
   fetcher: Fn
   initialData?: Data
+  cache: Cache
```

Comment:
shall we still keep the presets here to avoid less changes? it's easier for other js-runtime to customize swr